README.md

```diff
@@ -0,0 +1,29 @@
+# Lost
+
+Are you truly lost? Will you ever return home?
+A researcher on a space expedition is stranded on an alien world with no way to get home.  
+
+
+ ## How To Play
```

Comment:
Consider adding a note on that the structures (the small house and the puzzle structures to find when exploring) are generated during the game. So sometimes visiting a location twice can be useful. Also, - in case it's not in the book of secrets texts (I'm not sure about that) - mentioning that different times of day might help to find different puzzles might make sense.

You can use collapsible text (like in https://gist.github.com/joyrexus/16041f2426450e73f5df9391f7f7ae5f) to hide potential spoilers.

Reply:
ok will take note.